addon/templates/components/frost-modal-input.hbs

```diff
@@ -33,11 +33,15 @@
     minScrollbarLength=20
     maxScrollbarLength=150}}
     {{frost-bunsen-form
-      model=formModel
-      view=formView
-      value=formValue
+      bunsenModel=formModel
+      bunsenView=formView
+      inline=true
```

Comment:
I actually think we should probably just get rid of the inline property from bunsen as I don't know if we actually use it both ways and it makes the styles more complex.